reactorstate-maven-plugin/pom.xml

```diff
@@ -13,4 +13,87 @@
     <artifactId>reactorstate-maven-plugin</artifactId>
     <packaging>maven-plugin</packaging>
 
+    <prerequisites>
+        <maven>3.5</maven>
```

Comment:
prerequisites/maven is deprecated and has no use in Maven 3.0+, use _maven-enforcer-plugin_ instead:
https://maven.apache.org/pom.html#Prerequisites

Reply:
It has only been deprecated for projects, which are not of type `maven-plugin`.
In those cases Maven itself issues a warning at build-time.
I think it the documentation is just misleading.

Quote from [Prerequisites JavaDoc](http://maven.apache.org/ref/3.6.3/maven-model/apidocs/org/apache/maven/model/Prerequisites.html):
> getMaven()
> Get for a plugin project (packaging is maven-plugin), the minimum version of Maven required to use the resulting plugin.
In Maven 2, this was also specifying the minimum version of Maven required to build a project, but this usage is deprecated in Maven 3 and not checked any more: use the Maven Enforcer Plugin's requireMavenVersion rule instead.